src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -401,12 +387,30 @@ public static void TestingTooLargeSingleConversionToInfinity(string jsonString,
             {
                 if (json.TokenType == JsonTokenType.Number)
                 {
-                    Assert.True(json.TryGetSingle(out float floatValue));
-                    Assert.Equal(expectedFloat, floatValue);
+                    if (PlatformDetection.IsFullFramework)
+                    {
+                        Assert.False(json.TryGetSingle(out float _));
+
+                        try
+                        {
+                            json.GetSingle();
+                            Assert.True(false, $"Expected {nameof(FormatException)}.");
+                        }
+                        catch (FormatException)
```

Comment:
nit: collapse to single line (like we have elsewhere, for consistency):
```C
catch (FormatException) { }
```

Reply:
Updated to have `/* Expected exception */` as the contents, as that seems to be more consistent when there aren't try/catch chains.